Source/Device.swift.gyb

```diff
@@ -708,6 +714,10 @@ public enum Device {
     public var hasForceTouchSupport: Bool {
       return isOneOf(Device.allWatchesWithForceTouchSupport) || isOneOf(Device.allWatchesWithForceTouchSupport.map(Device.simulator))
     }
+  #elseif os(macOS)
+  public static var allMacs: [Device] {
+    return [.macBookProM2Max] // TODO: add others
+  }
```

Comment:
Please move macOS related code to a separate PR. Also needs to be generated using gyb.